packs/data/equipment.db/celestial-armor.json

```diff
@@ -68,41 +68,23 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Celestial Armor (vs. fiends)",
                 "predicate": {
                     "all": [
                         "origin:trait:fiend"
                     ]
                 },
-                "selector": "ac",
+                "selector": [
+                    "ac",
+                    "saving-throw"
+                ],
```

Comment:
Can we add two selectors at once like this?

Reply:
Yup, it's a relatively new thing that you can make the selector an array. see my other PR for condensing spell effect REs.